lib/identity/partialClient.js

```diff
@@ -54,25 +55,33 @@ class PartialClient extends CryptoConsumer {
           expiry_seconds: 1,
         },
       }
-      const request = await anonAuth.tsv1Fetch(
+      const response = await anonAuth.tsv1Fetch(
         `${this.config.apiUrl}/v2/storage/notes/challenge?id_string=${noteName}`,
         {
           method: 'PATCH',
           'Content-Type': 'application/json',
           body: JSON.stringify(body),
         }
       )
-      await checkStatus(request)
-      return true
-    } catch (_) {
-      return false
+      await checkStatus(response)
+      ret = {success: true, message: ""}
+      return ret
```

Comment:
unlike golang (😅), prefer immutability in js. you can just return the value directly.